docs/guides/1.3.0-upgrade-guide.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: "couchbase-capella"
+page_title: "Couchbase Capella Provider 1.3.0: Upgrade and Information Guide"
+sidebar_current: "docs-couchbase-capella-guides-130-upgrade-guide"
+description: |-
+Couchbase Capella Provider 1.3.0: Upgrade and Information Guide
+---
+
+# Couchbase Capella Provider 1.3.0: Upgrade and Information Guide
+
+Here is a list of what's new in 1.3.0
+
+## New Features
```

Comment:
@a-atri  @aniket-Kumar-c  What about system events and activity logs?

Reply:
@rajagp Added the system events and activity logs in the list